site/profile/manifests/proxy_puppet.pp

```diff
@@ -94,7 +94,7 @@
   haproxy::balancermember { 'foreman-backend8888':
     listening_service => 'foreman-frontend8888',
     server_names      => 'foreman.localdomain',
-    ipaddresses       => '172.28.128.22',
+    ipaddresses       => '172.28.128.20',
```

Comment:
Im not sure, maybe this was a typo or something? I found that to be an issue before I modified anything. But that front end is puppetmaster_webhook receiver, which should definitely be the Puppet master (172.28.128.20) :)